### DISCUSSION TITLE
Notifications don’t auto-scroll to the last read comment

### DISCUSSION THREAD

**Discussion by ironnysh** [community/community#131563] [Mobile]
I have a distinct feeling that this was possible in previous versions but I might be mistaken.
In any case, it would be great if notifications were automatically opened on the latest unread message.
This behavior is in line with the browser-based UX (mobile and desktop), which also includes the **Jump to bottom** button.

Thanks!

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.167.0 (134878402)
Device: iPhone SE (2nd generation) (iOS 17.6)
</details>

**Comment by stevepopovich** [comment]
Hi @ironnysh you are talking about when clicking on a notification from the notification inbox tab, correct? If so, yeah, the notification should scroll to the last place you read. If not, that is a bug. Can you reproduce this consistently?

**Reply by ironnysh** [reply]
Hi @stevepopovich, correct.

> Can you reproduce this consistently?

Yep, every day :-))

**Reply by stevepopovich** [reply]
Hmmm, I can't repro this. Could you possibly capture a video of this happening? Sorry for the inconvenience.

**Reply by ironnysh** [reply]
— @stevepopovich Sure. Here's an example (the file is too big for GitHub):
https://drive.proton.me/urls/6TG7GNAXTG#7ecixuXSOKuU

**Reply by stevepopovich** [reply]
Insane question—if you tap and then just wait a second, does it scroll then? Seems to be a delay for me in the scroll-to behavior

**Reply by ironnysh** [reply]
Nope. This slight delay happens in discussions—like this one—where the auto scroll works as expected.

**Reply by stevepopovich** [reply]
But in notifications, it isn't scrolling at all? Ok, lemme bring this to the team. Thank you!

**Reply by ironnysh** [reply]
That's correct.

Thanks a lot, @stevepopovich!

**Reply by stevepopovich** [reply]
Looks like this is a confirmed bug for some folks. We will put this in our backlog to fix. Thanks for reporting!

**Reply by ironnysh** [reply]
Awesome, thanks!

**Reply by Finkregh** [reply]
If that helps: same here, iOS 17.5.1. Ended up here as I was about to create an issue myself :)

**Reply by ironnysh** [reply]
Hi @stevepopovich, it looks like the auto-scroll is back to its normal, helpful state👌

I’m on the beta (18.4), but maybe @Finkregh can confirm if it’s fixed on 17.x as well.

**Reply by Finkregh** [reply]
I'm on 1.174.0, nothing changed
